demo/index.html

```diff
@@ -25,7 +25,7 @@
           Prism.highlightAll();
         });
     </script>
-    <script type="module" src="../src/auro-datetime.js"></script>
+    <script type="module" src="../index.js"></script>
```

Comment:
Please reference https://github.com/AlaskaAirlines/WC-Generator/blob/master/template/demo/index.html for all other changes.